_ont/ont-zte-f6005v3.md

```diff
@@ -0,0 +1,341 @@
+| Chipset      | ZTE ZX279133                                                      |
+| Flash        | 128 MB (SPI NAND FM25S01A)                                        |
+| RAM          | 128 MB                                                            |
+| System       | Customized Linux by ZTE                                           |
+| 2.5GBaseT    | No                                                                |
+| Optics       | SC/APC                                                            |
+| IP address   | 192.168.1.1                                                       |
+| Web Gui      | ✅ user `admin`, password `admin` or defined by ISP               |
+| SSH          |                                                                   |
+| Telnet       | ✅ [^1]                                                           |
+| Serial       | ✅ [^2]                                                           |
+| Form Factor  | ONT                                                               |
+
+{% include image.html file="f6005v3_tim_1.jpg" alt="F6005v3 TIM" caption="F6005v3 TIM" %}
+{% include image.html file="f6005v3_of_1.jpg" alt="F6005v3 OpenFiber" caption="F6005v3 OpenFiber" %}
+
+
+## List of software versions
+### HW V3.0 
+- V3.0.10P3N2 (OpenFiber) 
+- V3.0.10N06, internal version is V3.0.10P2N6 (TIM Italy)
+
+## List of partitions 
+
+### HW V3.0 
+
+| dev  | size     | erasesize | name             |
+| ---- | -------- | --------- | ---------------- |
+| mtd0 | 08000000 | 00020000  | "whole flash"    |
+| mtd1 | 00300000 | 00020000  | "uboot"          |
+| mtd2 | 00400000 | 00020000  | "others"         |
+| mtd3 | 00400000 | 00020000  | "parameter tags" |
+| mtd4 | 00400000 | 00020000  | "wlan"           |
+| mtd5 | 00800000 | 00020000  | "usercfg"        |
+| mtd6 | 00400000 | 00020000  | "middle"         |
+| mtd7 | 02a00000 | 00020000  | "kernel1"        |
+| mtd8 | 02a00000 | 00020000  | "kernel2"        |
+| mtd9 | 02500000 | 00020000  | "rootfs1"        |
+| mtd10 | 029e0000 | 00020000  | "rootfs2"       |
+
+
+This ONT supports dual boot, as visible from the presence of `kernel0` and `kernel1`, which contain the rootfs.
+The boot images can be swapped with the following command but currently not works because if U-Boot is updated, a revert will brick ONT:
+
+```sh
+upgradetest switchver X
+```
+
+Where `X` can be `0/1`, based on the image you want to boot from.
+
+
+You can also clone the currently running image into the other slot using this command:
+
+```sh
+syn_version
+```
+
+# Use
+{% include alert.html content="Commands have been tested on V3 HW rev. on OpenFiber firmwares" alert="Note"  icon="svg-info" color="blue" %}
+
+## Enable Telnet
+{% include alert.html content="This is an external script ([ZTE Telnet enabler](https://github.com/douniwan5788/zte_modem_tools)), use at your own risk! Credentials don't survive at reboot!" alert="Note"  icon="svg-info" color="blue" %}
```

Comment:
Do the GPON parameters (e.g. PLOAM) remain, though?

Reply:
> Do the GPON parameters (e.g. PLOAM) remain, though?

Yes setmac commands are permanent 😊

Reply:
Excellent!